src/framework/application.js

```diff
@@ -533,9 +533,10 @@ Object.assign(pc, function () {
             this.elementInput.app = this;
 
         this.vr = null;
-        // you can enable vr here, or in application properties
         if (options.vr) {
```

Comment:
Does this mean we need to bring back the Enable VR checkbox in the Editor and Platform?

Reply:
No, this isn't used by the application properties. But I will remove it.